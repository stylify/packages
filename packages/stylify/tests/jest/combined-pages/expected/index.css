.n{
	background: steelblue
}
.r{
	background: black
}
.e{
	color: #fff
}
.f{
	color: #888
}
.i{
	color: yellow
}
.s{
	color: darkred
}
.j{
	font-family: arial
}
.c{
	font-size: 123px
}
.g{
	font-size: 456px
}
.k{
	font-size: 12px
}
.m{
	font-size: 48px
}
.q{
	font-size: 14px
}
.d{
	line-height: 123px
}
.h{
	line-height: 456px
}
.p{
	margin: 0 auto
}
.o{
	max-width: 320px
}
.l{
	text-align: center
}

@media (min-width: 123px) {
.a{
	font-size: 64px
}
}

@media (max-width: 1250px) {
.b{
	font-size: 84px
}
}
