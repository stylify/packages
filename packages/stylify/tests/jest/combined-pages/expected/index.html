<!-- stylify-pregenerate font-size:123px line-height:123px color:#fff color:#888 /stylify-pregenerate -->
<!--
stylify-screens
	'testScreen': '(min-width: 123px)',
	'dynamic\\w+': ({ match }) => `(max-width: ${match.getCapture(0).replace('dynamic', '')})`
/stylify-screens

stylify-pregenerate font-size:456px line-height:456px color:yellow /stylify-pregenerate
-->

<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title>Document</title>
</head>
<body class="j k">
	<div class="
		l
		m
		n
		o
		p
		e
		a
		b
	">
		Index Page
	</div>
</body>
</html>
